.buildkite/bk.fips-integration.pipeline.yml

```diff
@@ -0,0 +1,217 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
+  ASDF_MAGE_VERSION: 1.14.0
+  FIPS: "true"
```

Comment:
I'd recommend reducing the scope of this variable at least to the step level. Currently, it's hard to understand what step is affected by this variable